jvm-libs/linea/web3j-extensions/src/main/kotlin/net/consensys/linea/web3j/DomainObjectMappers.kt

```diff
@@ -133,11 +134,5 @@ fun EthBlock.TransactionObject.toBytes(): Bytes {
     }
     .build()
 
-  if (signature != null) {
-    if (this.v != transaction.v.toLong()) {
```

Comment:
Besu V24 transaction.v was returning null for non-frontier tx, hence removed this check.